cmd/fyne/internal/commands/build.go

```diff
@@ -331,6 +332,12 @@ func injectMetadataIfPossible(runner runner, srcdir string, app *appData,
 		return nil, nil
 	}
 
+	fyneGoModVersionConstraintOlderThan203 := version.NewConstrainGroupFromString(">=2.3")
+	if fyneGoModVersionConstraintOlderThan203.Match(fyneGoModVersionNormalized) {
+		app.VersionOlderThan203 = true
```

Comment:
I guess you mean `VersionAtLeast23` not `VersionOlderThan203`?

Reply:
Given that it is just an identifier and not a int why not `VersionAtLeast2_3` which would allow 2_11 if required

Reply:
Indeed, switched to VersionAtLeast2_3.